### PR TITLE
update build number to current status

### PR DESCRIPTION
**Pull request by jessyrod68**
will this change also change the name in the menu on the right? see attachment for reference.

### CODE REVIEW

**Comment by jessyrod68** [comment]
What's new with 1903

**Comment by jessyrod68** [comment]
*left

**Comment by mypil** [comment]
@nenonix - please share your thoughts on this and kindly do the copy/edit review. Thanks.

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: wip/at-home/Whats-new-wip-at-home-19h1.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ ms.topic: article
 ms.prod: w10
 ---
 
-# What's new for Windows 10 Insider Preview Builds (19H1)
+# What's new for Windows 10 Insider Preview Builds 1903
 The [Windows Insider Program](https://insider.windows.com/en-us/) lets you preview builds of the upcoming release of Windows 10. This topic lists all new Windows 10 features for you to try.
```
Comment (nenonix) [inline wip/at-home/Whats-new-wip-at-home-19h1.md]:
```suggestion
The [Windows Insider Program](https://insider.windows.com) lets you preview builds of the upcoming release of Windows 10. This topic lists all new Windows 10 features for you to try. 
```

**Pull request merged**